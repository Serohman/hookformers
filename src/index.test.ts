import * as publicApi from "./index";

describe("Public API", () => {
  it("should not change without a deliberate update", () => {
    expect(publicApi).toMatchSnapshot();
  });
});
