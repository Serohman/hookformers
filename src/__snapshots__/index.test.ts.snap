// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Public API should not change without a deliberate update 1`] = `
{
  "useTextClassification": [Function],
}
`;
